### DISCUSSION TITLE
Swap between List-view (details) and Album-art-view (tiles)

### DISCUSSION THREAD

**Discussion by jalsco** [jcorporation/myMPD#938] [Q&A]
myMPD version: 10.1.6

Is there some way to swap between - Viewing the Detail list and viewing the Album cover tiles ?  
I see only the List view when in Queue and when in "browse file system"
I see only the album-cover tiles view - when in "browse database". 

I love both views - and I like to use both, i.e. swap between them - at different times, depending on what I am doing or searching for.  I hope doing that is possible. If so, it is not obvious to me how to do it.  

Is there some "view setting" that I am not seeing ? 

and also - in Database - can currently browse by : Album, Albumartist, Artist, and Genre ... please , please,  add "Song" or "Title" to those options.  😁👍

**Comment by jcorporation** [comment]
Additional browse tags can be enabled in the settings.

Other question: I think I do not understand your question.

**Reply by jalsco** [reply]
you do not understand the question ? 

I like seeing all the graphic Tiles - for each song, not just for each Album...  but I believe that option to view, only exists in "Browse Database".  When  "browsing filesystem" or "viewing Queue" -  it seems like the only display option is to view the TEXT list of the songs.   

IS that easier to understand ?

**Reply by jcorporation** [reply]
Ah, Better ;) The album cover grid only exists for albums. The other views are text only.
